.github/workflows/ci.yml

```diff
@@ -230,3 +232,28 @@ jobs:
       - uses: actions/checkout@v4
       - name: Verify the committed UI artifact matches the published one in restate-web-ui
         run: ./tools/scripts/verify-ui-artifact
+
+  jepsen:
+    needs: docker
+    # id tokens are not available on forks
+    if: github.event.repository.fork == false
+    runs-on: warp-ubuntu-latest-arm64-4x
+    name: Run Jepsen tests
+    concurrency:
+      group: singleton # due to the shared Jepsen worker cluster across PRs
```

Comment:
Is it intended that other pending jobs will be canceled if a new concurrent job gets started? I am looking at this description: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/control-the-concurrency-of-workflows-and-jobs#using-concurrency-in-different-scenarios.

Reply:
Will canceled jobs be treated as a failure or success for an affected PR?

Reply:
It is absolutely not, let me think about how better to organize this.